src/api.ts

```diff
@@ -159,4 +171,35 @@ export function createMongoDBOIDCPlugin(
 }
 
 /** @internal */
-export const kDefaultOpenBrowserTimeout = 10_000;
+export const kDefaultOpenBrowserTimeout = 20_000;
```

Comment:
I’m bumping this a bit since I didn’t realize that this time isn’t being affected by `serverSelectionTimeoutMS`, i.e. we truly have 5 minutes for completing authentication.

(This is only the time for waiting for the browser to open a local URL, not the time to successful auth)